src/plugins/identity/contractIdentities.test.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import {node} from "../../core/graphTestUtil";
+import {Graph, NodeAddress} from "../../core/graph";
+import * as Weights from "../../core/weights";
+import {_contractWeightedGraph} from "./contractIdentities";
+
+describe("plugins/identity/contractIdentities", () => {
+  const a = node("a");
+  const b = node("b");
+  const c = node("c");
+  const graph = () => new Graph().addNode(a).addNode(b);
```

Comment:
Nit: prefer including an edge contraction, though OK with accepting as-is to delegate this to `Graph.contractNodes()` tests.

Reply:
Yeah, this is just a sanity check that we've delegated to Graph.contractNodes--which has thorough testing of edge contractions.